### PR TITLE
Add photo video

### PR DESCRIPTION
**Pull request by Sushma7785**
- Added capture picture functionality and UI
- Added video recording
- Added camera mode listener
- Maintain connection status in app when callback is received from connection listener

### CODE REVIEW

File: app/src/main/java/com/yuneec/example/component/actitivty/MainActivity.java [hunk 1]
```diff
@@ -148,7 +149,10 @@ public void publishCameraResult(final String result) {
 
             @Override
             public void run() {
-                Toast.makeText(getApplicationContext(), result, Toast.LENGTH_LONG).show();
+                Log.d(TAG, result);
+                if (!result.equals("Success")) {
+                    Toast.makeText(context, result + "-" + "Please make sure SD card is inserted", Toast.LENGTH_LONG).show();
```
Comment (julianoes) [inline app/src/main/java/com/yuneec/example/component/actitivty/MainActivity.java]: Doesn't `Toast.LENGTH_LONG` feel to long? I think I prefer the shorter notes.

File: app/src/main/java/com/yuneec/example/component/fragment/CameraSettingsFragment.java [hunk 2]
```diff
@@ -53,6 +58,12 @@
 
     Camera.ShutterSpeedS shutterSpeedS = new Camera.ShutterSpeedS();
 
+    Button capturePicture;
+
+    Button video;
+
+    private Camera.Mode cameraMode = Camera.Mode.PHOTO;
```
Comment (julianoes) [inline app/src/main/java/com/yuneec/example/component/fragment/CameraSettingsFragment.java]: Do you know how it is initialized? What about starting with `UNKNOWN`?

File: app/build.gradle [hunk 3]
```diff
@@ -31,5 +31,5 @@ dependencies {
     })
     compile 'com.android.support:appcompat-v7:25.3.1'
     testCompile 'junit:junit:4.12'
-    compile 'com.github.YUNEEC:Yuneec-SDK-Android:v0.5.8'
+    compile 'com.github.YUNEEC:Yuneec-SDK-Android:v0.6.0'
```
Comment (julianoes) [inline app/build.gradle]: Please go to `v0.6.1` while you're at it.

File: app/src/main/java/com/yuneec/example/component/fragment/CameraSettingsFragment.java (hunk 2 above)
Comment (Sushma7785) [reply app/src/main/java/com/yuneec/example/component/fragment/CameraSettingsFragment.java]: Yes makes sense. Will change it.

**Pull request merged**